src/test/kotlin/com/hibob/academy/dao/PetDaoTest.kt

```diff
@@ -116,10 +93,77 @@ class PetDaoTest @Autowired constructor(private val sql: DSLContext){
         assertEquals(expectedCount, actualCount)
     }
 
+    @Test
+    fun `update pets name`(){
+        val pet = PetCreationRequest("Kaia", PetType.convertStringToPetType("Dog"), companyId, Date.valueOf("2024-09-13"), ownerId)
+        val petId = petDao.createPet(pet)
+        val newName = "Charlie"
+        petDao.updatePetName(petId, newName, companyId)
+        val expected = PetData(petId, newName, pet.type, pet.companyId, pet.dateOfArrival, pet.ownerId)
+        val updatedPet = petDao.getPetById(petId, companyId)
+        assertEquals(expected, updatedPet)
+    }
+
+    @Test
+    fun `delete a pet`(){
+        val pet = PetCreationRequest("Max", PetType.convertStringToPetType("Cat"), companyId, Date.valueOf("2024-09-13"), ownerId )
+        val petId = petDao.createPet(pet)
+        petDao.deletePet(petId, companyId)
+        assertEquals(null, petDao.getPetById(petId, companyId))
+    }
+
+    @Test
+    fun `adopt multiple pets`(){
+        val listOfPetsIds = mutableListOf<Long>()
+        val ownerIdThatAdopts = 20L
+
+        val pet1 = PetCreationRequest("Angie", PetType.convertStringToPetType("DOG"), companyId, Date.valueOf("2010-05-20"), 18L)
+        val id1 = petDao.createPet(pet1)
+
+        val pet2 = PetCreationRequest("Nessy", PetType.convertStringToPetType("DOG"), companyId, Date.valueOf("2010-05-20"), ownerId)
```

Comment:
again - more readable with null and not val which is null - in this case ownerId
It makes it easier to understand what 1 failed and other 2 pass